### ISSUE TITLE
smoothHeight height "miscalculates height", covers text.

### ISSUE THREAD

**Issue by bfrisch1** [woocommerce/FlexSlider#1608]
Hi there!

Having, what I hope is, a minor issue with FlexSlider.

You can see the issue here: https://thefreshplan.com/menu-test-page/

The first slide is fine, but pay attention when you click next slide, you can see the smoothHeight, actually covers up the bottom 20%ish of the slide.

Here's my settings if that helps (sorry about the weird formatting, had issues with insert-code 'ticks':

```
`(function($) {
    $(window).load(function() { 

```

```
   $('.flexslider-test').flexslider({
        animation: 'slide',
        slideshow: false,
        controlsContainer: $(".custom-controls-container"),
        customDirectionNav: $(".custom-navigation a"),
        pauseOnHover: true,
        controlNav: false,
        direction: 'horizontal',
        smoothHeight: 'true',
        itemWidth: 300,
        itemMargin: 50,
        minItems: 1,
        maxItems: 2,
        startAt:0,
    });
});`
```

thanks in advance!

**Comment by bfrisch1** [comment]
Can anyone chime in here please? Thx

**Comment by bfrisch1** [comment]
I did leave a ticket with woocommerce and they we're sort of helpful. But it didn't fix my issue. Here is their response below:

We don't actually support FlexSlider itself here, but we do support the paid version WooSlider. I've seen a similar issue with heights before (in WooSlider) which is due to the zindex display attributes.

For example, these two lines in jquery.flexslider.js were changed from

 -            `slider.slides.eq(slider.currentSlide).css({"zIndex": 1}).animate({"opacity": 0}, slider.vars.animationSpeed, slider.vars.easing);`
 -            `slider.slides.eq(target).css({"zIndex": 2}).animate({"opacity": 1}, slider.vars.animationSpeed, slider.vars.easing, slider.wrapup);`
to

 +            `slider.slides.eq(slider.currentSlide).css({"zIndex": 1, "display": "none"}).animate({"opacity": 0}, slider.vars.animationSpeed, slider.vars.easing);`
 +            `slider.slides.eq(target).css({"zIndex": 2, "display": "block"}).animate({"opacity": 1}, slider.vars.animationSpeed, slider.vars.easing, slider.wrapup);`
